src/renderer/components/dialogs/Settings.tsx

```diff
@@ -115,6 +116,9 @@ export default function Settings(props: DialogProps) {
       (await DeltaBackend.call('settings.setDesktopSetting', key, value)) ===
       true
     ) {
+      if (key === 'minimizeToTray') {
+        ipcBackend.send('updateTrayIcon')
```

Comment:
Can we refactor this into a `DeltaBackend.call('tray.updateTrayIcon')` ? @Simon-Laux what's your opinion on this?

Reply:
remember that we want to weaken the connection between electron and core?
Please create a function under the Runtime interface and let it do the ipc call.